package.json

```diff
@@ -32,24 +32,25 @@
     "url": "http://blog.millermedeiros.com/"
   },
   "devDependencies": {
-    "esprima": "1.0",
-    "mocha": "https://github.com/millermedeiros/mocha/tarball/latest",
     "chai": "1.4",
+    "esprima": "1.0",
     "glob": "3.1",
     "istanbul": "~0.1.23",
     "jshint": "~2.3.0",
+    "mocha": "https://github.com/millermedeiros/mocha/tarball/latest",
```

Comment:
since it's only used during `dev`, I don't think it's a big problem to have it linked to a github repository. but I would be open to a separate package if you think it would improve things.